### DISCUSSION TITLE
Transpile python to carbon as a cython alternative or something like Pyo3?

### DISCUSSION THREAD

**Discussion by gertcuykens** [carbon-language/carbon-lang#1712] [General]
How much work would it take to make a python to carbon transpiler like cython or something like Pyo3?

https://rustrepo.com/repo/PyO3-pyo3
https://cython.readthedocs.io/en/latest/src/quickstart/overview.html

**Comment by MichaelBonnet** [comment]
This sort of thing is quite difficult to quantify. There's many determining variables - number of contributors, time dedicated per contributor, etc.

**Reply by gertcuykens** [reply]
If you would describe it in a roadmap, what steps needs to be done or can be forked over from cython? For example do we get from other projects or from python itself a AST for free so we don't have to parse python code and start from there? I assume allot of cython work can be salvaged or am I missing something?
